src/Pages/Edit.elm

```diff
@@ -128,7 +128,7 @@ update msg model =
                             { info | socials = newSocials }
                     in
                     ( { model | info = Api.Success newInfo }
-                    , Effect.none
```

Comment:
currently, the UpdateField message is all over the place, it could be possible to merge it, not going to be a big deal though so don't spend 3 hours on this one thing ;)